test/integration/model.test.js

```diff
@@ -1583,6 +1583,20 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         });
       });
     });
+    it('returns multiple rows when using group', function(done) {
+      var self = this;
+      this.User.bulkCreate([{username: 'user1', data: 'A'},
+                           {username: 'user2', data: 'A'},
+                           {username: 'user3', data: 'B'}]).success(function() {
```

Comment:
`.succes()` is deprecated, please use `then()` and return `count` to main the promise chain, you can also return the entire promise chain rather than use `done`